src/components/stepindicator/StepIndicator/StepIndicator.tsx

```diff
@@ -16,6 +16,8 @@ type StepIndicatorProps = {
     HTMLHeadingElement
   >
   headingLevel: HeadingLevel
+  stepText?: string
```

Comment:
If there are more of these, would it be better to do an object with multiple entries?
```
text: { step?: string, of?: string }
```
That way we can extend without adding loads more props?

Reply:
These are the only two strings on the top-level component, so would it still be beneficial to put it in such an object? 

The other strings are on the StepIndicatorStep sub-component, which _could_ use such a treatment, but I don't think we have that pattern on other components and wanted to keep this PR tight enough that we can unblock the project requirement.

Reply:
Okay no worries, that works. Lemme test.